### ISSUE TITLE
pip not executable

### ISSUE THREAD

**Issue by Wissperwind** [indygreg/python-build-standalone#82]
Hi,
I can execute ./python3 cool!
Python works. But pip does not. I can't execute ./pip3
```

osboxes@osboxes:~/Documents/resource/python/install/bin$ ./pip3
bash: ./pip3: /build/out/python/install/bin/python3: bad interpreter: No such file or directory

```

**Comment by indygreg** [comment]
This behavior is documented at https://python-build-standalone.readthedocs.io/en/latest/quirks.html#bad-shebangs-in-python-scripts.

#61 purports to fix this. But I've been hesitant to commit because I'm unsure how portable that shebang trick is.

**Comment by indygreg** [comment]
This should be fixed as of commit 92fe187a43f77a6a5e03581f4944225b94b1b328. It will make it in the next release.
